addons/website/static/src/js/theme_preview_kanban.js

```diff
@@ -0,0 +1,33 @@
+odoo.define('website.theme_preview_kanban', function (require) {
+"use strict";
+
+var KanbanController = require('web.KanbanController');
+var KanbanView = require('web.KanbanView');
+var ViewRegistry = require('web.view_registry');
+
+var ThemePreviewKanbanController = KanbanController.extend({
+    // -------------------------------------------------------------------------
+    // Public
+    // -------------------------------------------------------------------------
+
+    /**
+     * @override
+     */
+    start: function () {
+        this.$el.addClass('o_view_kanban_theme_preview_controller');
+        return this._super.apply(this, arguments);
+    },
+});
+
+var ThemePreviewKanbanView = KanbanView.extend({
+    searchMenuTypes: [],
```

Comment:
So this was the way to have no searchview ?

Reply:
no, but it avoid to compute the filter, gorup by button.
So we dont compute something hide in css later